<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>学生信息管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icon.jpg</normaloff>:/icon.jpg</iconset>
  </property>
  <property name="toolTip">
   <string>这是学生管理系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(70, 98, 126);
color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="4">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 700 20pt &quot;Microsoft YaHei UI&quot;;</string>
         </property>
         <property name="text">
          <string>学生信息管理系统</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/user.jpg);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="show_username">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 15pt &quot;楷体&quot;;</string>
            </property>
            <property name="text">
             <string>text</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 95, 32);
</string>
            </property>
            <property name="text">
             <string>退出</string>
            </property>
            <property name="btn" stdset="0">
             <string>main</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(67, 94, 121, 255);
font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(230, 230, 230);</string>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="5">
         <widget class="QPushButton" name="btn_update">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	color: white;
	background-color: #47af9a;
}

QPushButton:pressed {
   background-color: #34495e;
}</string>
          </property>
          <property name="text">
           <string>修改</string>
          </property>
          <property name="btn" stdset="0">
           <string>main</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="styleSheet">
           <string notr="true">color: #34495e;
</string>
          </property>
          <property name="text">
           <string>全选</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_cnt">
          <property name="styleSheet">
           <string notr="true">color: red;
</string>
          </property>
          <property name="text">
           <string>Label_cnt</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QPushButton" name="btn_del">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	color: white;
	background-color: #47af9a;
}

QPushButton:pressed {
   background-color: #34495e;
}</string>
          </property>
          <property name="text">
           <string>删除</string>
          </property>
          <property name="btn" stdset="0">
           <string>main</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="btn_add">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	color: white;
	background-color: #47af9a;
}

QPushButton:pressed {
   background-color: #34495e;
}</string>
          </property>
          <property name="text">
           <string>增加</string>
          </property>
          <property name="btn" stdset="0">
           <string>main</string>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QLineEdit" name="le_search">
          <property name="styleSheet">
           <string notr="true">border : 2px solid #bdc3c7;
border-radius: 6;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="8">
         <widget class="QPushButton" name="btn_search">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	color: white;
	background-color: #47af9a;
}

QPushButton:pressed {
   background-color: #34495e;
}</string>
          </property>
          <property name="text">
           <string>搜索</string>
          </property>
          <property name="btn" stdset="0">
           <string>main</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="9">
         <widget class="QTableWidget" name="tableWidget">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #34495e;</string>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="pushButton_5">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 95, 32);
</string>
          </property>
          <property name="text">
           <string>清空学生</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
